### ISSUE TITLE
Feature Request: Ability to add message/handler when retry is triggered.

### ISSUE THREAD

**Issue by fmaiabatista** [softonic/axios-retry#134]
Hi, I took a look at the API but it doesn't seem to include any way for us to configure a message when the retry happens.

I was wondering if it'd be possible to add a way that we can set a message/handler whenever the retry is triggered.

Something like:

```
const handler = () => console.log("Retrying...");

axiosRetry(axios, {
  retries: 5,
  retryDelay: axiosRetry.exponentialDelay,
  onRetry: handler
});
```

**Comment by mawrkus** [comment]
HI!
If it's only for logging a message; I guess you can use a customised `retryDelay` (as a function) or `retryCondition`.

**Comment by fmaiabatista** [comment]
Alright! I'll try that! Thanks!

**Comment by mriedem** [comment]
Was this fixed with #202?
